src/elements/common/item/IconCell.js

```diff
@@ -20,13 +21,29 @@ import './IconCell.scss';
 type Props = { dimension?: number, intl: IntlShape, rowData: BoxItem };
 
 const IconCell = ({ intl, rowData, dimension }: Props) => {
-    const { type, extension, has_collaborations, is_externally_owned }: BoxItem = rowData;
+    const { type, extension, has_collaborations, is_externally_owned, archive_type }: BoxItem = rowData;
     let title;
+    const is_archive = archive_type === 'archive';
+    const is_archived_content = archive_type === 'archive-content';
     switch (type) {
         case TYPE_FILE:
             title = intl.formatMessage(messages.file);
             return <FileIcon dimension={dimension} extension={extension} title={title} />;
         case TYPE_FOLDER:
+            if (is_archive) {
+                return (
+                    <Archive aria-label={intl.formatMessage(messages.archive)} height={dimension} width={dimension} />
+                );
+            }
+            if (is_archived_content) {
+                return (
+                    <FolderArchive
+                        aria-label={intl.formatMessage(messages.archivedFolder)}
+                        height={dimension}
+                        width={dimension}
+                    />
+                );
+            }
```

Comment:
At first I tried adding those icons to `<FolderIcon>`, but blueprint icons don't have a title and aria-label is not compatible with `React.ReactElement<string>` type.